application/src/main/java/org/togetherjava/tjbot/routines/ModAuditLogRoutine.java

```diff
@@ -363,4 +362,4 @@ String getVerb() {
             return verb;
         }
     }
-}
```

Comment:
dont we have a consistent line ending setting? the never ending story of line ending diffs 😆